library/Core.cpp

```diff
@@ -1249,7 +1249,14 @@ command_result Core::runCommand(color_ostream &con, const std::string &first_, s
         }
         else if (res == CR_NEEDS_CONSOLE)
             con.printerr("%s needs an interactive console to work.\n"
-                          "Please run this command from the DFHack terminal.\n", first.c_str());
+                          "Please run this command from the DFHack terminal.\n\n"
```

Comment:
Mind if I change this to `console` for consistency?

Reply:
Sgtm